### ISSUE TITLE
When creating a PSACT value as YAML, you are not redirected back to the `Pod Security Admissions` tab

### ISSUE THREAD

**Issue by markusewalker** [rancher/dashboard#8057]
**Is your feature request related to a problem? Please describe.**

When creating a new PSACT value, when attempting to do so while editing as YAML, the following behavior happens:

![Jan-27-2023 11-26-54](https://user-images.githubusercontent.com/52591819/215179826-6ffc1ff8-553b-4dd9-b9a0-62c98a3c3bba.gif)

As seen in the above video, the new PSACT value creates, but rather than go back to the `Pod Security Admissions` tab and show all of the created templates, it simply says `Saved` and stays on the same edited YAML screen. Additionally, clicking `Create` again gives the expected error shown as the value is already created.

**Describe the solution you'd like**

If a user is creating a PSACT value via YAML, once it has been successfully created, they should be redirected to the `Pod Security Admissions` tab.

**Comment by gaktive** [comment]
Per @richard-cox, we're missing a list location on the modal to know where to go back to.

**Comment by richard-cox** [comment]
It's better to ensure the model dictates where it's list is

shell/models/management.cattle.io.podsecurityadmissionconfigurationtemplate.ts
```
  get listLocation() {
     return ...location of list
  }
```
If this doesn't work, it may need to also be used by
```
  get doneOverride() {
    return this.listLocation;
  }
```

If all else fails  shell/edit/management.cattle.io.podsecurityadmissionconfigurationtemplate.vue
```
<CruResource :done-route="<list location>">
```

**Comment by cnotv** [comment]
@richard-cox FYI it worked with `doneOverride()`, although I had to convert the TS file into JS.

**Comment by cnotv** [comment]
The issue seems related to the side nav configuration as it complains afterwards to miss something in the `product` config. We'll have to add it to the documentation as well.

**Comment by markusewalker** [comment]
Validated that this is addressed on `v2.7-head` https://github.com/rancher/rancher/commit/ab561bf837ca6051176bf169784edb01d9dee134. See results below:

**ENVIRONMENT**
- Rancher install: Helm
- Rancher version: `v2.7-head` https://github.com/rancher/rancher/commit/ab561bf837ca6051176bf169784edb01d9dee134
- Browser: Chrome

**TEST RESULTS**
|Scenario|Test Case|Results|
|---|---|---|
|1|Add new PSA value via YAML|:white_check_mark:|
|2|Add new PSA value via YAML and leave the versioning blank|:white_check_mark:|

**Scenario 1**
1. As an admin user, navigated to Cluster Management > Advanced > Pod Security Admissions.
2. Clicked `Create`.
3. Edited via YAML and changed enforce , warning and audit  to `rancher-baseline` and created.
4. Validated that the this new PSA successfully created via the YAML option.
______
**Scenario 2**
1. As an admin user, navigated to Cluster Management > Advanced > Pod Security Admissions.
2. Clicked `Create`.
3. Edited via YAML and change enforce , warning and audit  to `rancher-baseline-no-version`.
     - Left audit-version , enforce-version  and warning-version all blank.
4. Validate that this new PSACT successfully creates.
